### ISSUE TITLE
facts, action guides, campaign runs, and the homepage shouldn't show up in search results

### ISSUE THREAD

**Issue by mikefantini** [DoSomethingArchive/legacy-website#3206]
we're currently having individual facts show up in search results. These should be removed from possible results (user gets 404'd when they try to access the pages).

Fact pages and campaigns that have facts on them however should remain.

![image](https://cloud.githubusercontent.com/assets/6330971/4722525/ccc25780-5940-11e4-8f0f-4a070062d45c.png)

The homepage is also showing up and shouldn't:

![image](https://cloud.githubusercontent.com/assets/6330971/4722677/2e05403e-5941-11e4-9e66-c0119bc3d1b9.png)

And action guides are showing up too but shouldn't be:

![image](https://cloud.githubusercontent.com/assets/6330971/4722684/418f1774-5941-11e4-9588-9e19f81b66ba.png)

Neither should campaign runs:

![image](https://cloud.githubusercontent.com/assets/6330971/4723802/f9228bce-594a-11e4-9ff1-fdc92d71117d.png)

or images:

![image](https://cloud.githubusercontent.com/assets/6330971/4909036/0327c8ca-6470-11e4-8bc0-513f5885a3b2.png)

basically the only thing I should be able to click-through to from the search results are:
- campaigns
- campaign collections
- fact pages
- static content pages
- taxonomy views (only if published)

@blisteringherb made some updates to this, let me know if you have any questions

**Comment by mikefantini** [comment]
ideally the content on an action guide would get associated with the campaign(s) it is associated with and help that campaign's search rank

**Comment by mikefantini** [comment]
Ideally the content on a campaign run would get associated with the campaign it is associated with and help that campaign's search rank.

**Comment by mikefantini** [comment]
still seeing fact about homelessness as first result when searching homelessness

![image](https://cloud.githubusercontent.com/assets/6330971/4945745/e6337678-660a-11e4-983b-c698732edb37.png)

**Comment by mikefantini** [comment]
this seems to be resolved. closing this issue
